Ray.jl/src/runtime.jl

```diff
@@ -98,8 +102,10 @@ function task_executor(ray_function)
                             get_current_job_id())
     # for some reason, `eval` gets shadowed by the Core (1-arg only) version
     # func = Base.eval(@__MODULE__, Meta.parse(rayjll.CallString(fd)))
-    @info "Calling $func"
-    return func()
+    args = map(arg -> String(take!(GetData(arg))), ray_objects)
+    arg_string = join(string.("::", typeof.(args)), ", ")
+    @info "Calling $func($arg_string)"
```

Comment:
```julia
julia> Ray.task_executor(length, [ray_obj])
calling delete on: 0x10656e740
[ Info: Calling length(::String)
5
```

Reply:
lol - this was to show an example of the `info` message, I still haven't got the function working 🙈

Reply:
hey we gotta take the wins where we can